Packs/ExpanseV2/Scripts/ExpanseAggregateAttributionCI/ExpanseAggregateAttributionCI_test.py

```diff
@@ -5,13 +5,15 @@
 
 INPUT = [
     {
-        "name": "server 1",
-        "sys_id": "1234",
-        "sys_class_name": "cmdb_ci",
-        "asset": {
-            "display_value": "server 1",
-            "link": "https://servicenow.com/5678",
-            "value": "5678"
+        "Attributes": {
```

Comment:
What caused this change in the response object? how come we didn't catch it when running the playbooks?

Reply:
It looks like I was missing the `Attributes` object earlier. A previous version of the attribution playbook had passed that object, rather than it's parent to the ExpanseAggregationAttributionCI script.